File: chapter3/createRDD.scala
// Initializing Spark in Scala
import org.apache.spark.SparkConf
import org.apache.spark.SparkContext
import org.apache.spark.SparkContext._

val conf = new SparkConf().setMaster("local").setAppName("My App")
val sc = new SparkContext("local", "My App")
//Create RDD by loading external dataset
val lines = sc.textFile("README.md")
//Create RDD by parallelize the collection in a driver program
val lines = sc.parallelize(list("pandas", "i like pandas"))

File: chapter3/createRDD.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#########################################################################
# File Name: createRDD.py
# Author: lpqiu
# mail: qlp_1018@126.com
# Created Time: Fri Nov 14 08:36:38 2014
#########################################################################
from pyspark import SparkConf,SparkContext


conf = SparkConf().setMaster("local").setAppName("My App")
sc = SparkContext(conf)
#Create RDD by loading external dataset
lines = sc.textFile("README.md")
#Create RDD by parallelize the collection in a driver program
lines = sc.parallelize(["pandas", "i like pandas"])

